packages/react/src/contents/hooks/useSeoMetadata.ts

```diff
@@ -0,0 +1,92 @@
+import { buildLinks, buildMetas, getDefaultAppLinks } from '../utils';
+import {
+  fetchSEO,
+  getSEO,
+  getSEOError,
+  isSEOFetched,
+  isSEOLoading,
+  StoreState,
+} from '@farfetch/blackout-redux';
+import {
+  GetSEOQuery,
+  SeoPageType,
+  SeoSubPageType,
+} from '@farfetch/blackout-client';
+import { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import useAction from '../../helpers/useAction';
+import type {
+  AppIconLinks,
+  Link,
+  Meta,
+  UseSeoMetadata,
+  UseSeoMetadataOptions,
+} from '../types';
+
+/**
+ * Hook to return seo metadata to populate document head.
+ *
+ * @param data - object that receives SEO query, icon links, additional links, metas and options
+ *
+ * @returns - Returns actions and SEO metadata state for a given path.
+ */
+const useSeoMetadata = ({
+  query,
+  appIconLinks = getDefaultAppLinks(),
+  links = [],
+  metas = [],
+  options = {},
```

Comment:
To be consistent with the other hooks design, can you move the options `property` to the second parameter?